yaml-cpp.spec

```diff
@@ -0,0 +1,26 @@
+### RPM external yaml-cpp 0.7.0
+
+Source: https://github.com/jbeder/yaml-cpp/archive/refs/tags/%{n}-%{realversion}.tar.gz
+
+BuildRequires: cmake ninja
+
+%prep
+%setup -n %{n}-%{realversion}
```

Comment:
@forthommel , please use `%{n}-%{n}-%{realversion}`

Reply:
Indeed, sorry for the false start!

Reply:
@smuzaffar I also updated the `cmake` source path. I guess this is also needed, right?